src/model/object/entity/individual/fred/fred.cpp

```diff
@@ -11,10 +11,14 @@ Fred::Fred(long ID, Point position, double a, double energy) :
     memory.insert({TURN_RIGHT, 1});
 }
 
+Fred * Fred::reproduce(long ID) {
+    double energy = getEnergy();
+    updateEnergy(1 - REPRODUCTION_CHILD_FRACTION);
```

Comment:
`-`?

Reply:
the child gets REPRODUCTION_CHILD_FRACTION (like, 20% or so) energy, so
the parent gets 1 - REPRODUCTION_CHILD_FRACTION (80%) energy